.github/workflows/srt_nuopc.yml

```diff
@@ -0,0 +1,195 @@
+# CIME scripts regression tests
+
+name: nuopc scripts regression tests
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, maint-5.6 ]
```

Comment:
I think that we don't need (or want) cmeps tests on maint-5.6

Reply:
Does removing references to maint-5.6 include srt.yml?